File: pascals.cpp
#include <iostream>
using namespace std;

// Функция для перевода давления из паскалей в атмосферы
double pascalToAtm(double pascals) {
    return pascals / 101325;
}

int main() {
    double pascals;
    cout << "Введите давление в паскалях: ";
    cin >> pascals; // Получение ввода от пользователя

    double atmospheres = pascalToAtm(pascals); // Вызов функции для конвертации
    cout << "Давление в атмосферах: " << atmospheres << " атм" << endl;

    return 0;
}

File: pascals.py
def pascal_to_atm(pascals):
    """
    Функция для перевода давления из паскалей в атмосферы.
    """
    atmospheres = pascals / 101325
    return atmospheres

# Пример использования функции
pascals = float(input("Введите давление в паскалях: "))
atmospheres = pascal_to_atm(pascals)
print(f"Давление в атмосферах: {atmospheres:.2f} атм")